<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vvMainWindow</class>
 <widget class="QMainWindow" name="vvMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>645</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="vvResources.qrc">
    <normaloff>:/vvResources/Icons/logo.png</normaloff>:/vvResources/Icons/logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1070</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>&amp;Recent Files</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionClear_Menu"/>
    </widget>
    <widget class="QMenu" name="menuSaveAs">
     <property name="title">
      <string>Save &amp;As</string>
     </property>
     <addaction name="actionSavePCAP"/>
     <addaction name="actionSaveCSV"/>
     <addaction name="actionSavePositionCSV"/>
     <addaction name="actionSavePCD"/>
     <addaction name="actionSavePLY"/>
     <addaction name="actionSaveLAS"/>
    </widget>
    <widget class="QMenu" name="menu_Open">
     <property name="title">
      <string>&amp;Open</string>
     </property>
     <addaction name="actionOpenPcap"/>
     <addaction name="actionOpen_Sensor_Stream"/>
     <addaction name="separator"/>
     <addaction name="actionChoose_Calibration_File"/>
    </widget>
    <addaction name="menu_Open"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="separator"/>
    <addaction name="menuSaveAs"/>
    <addaction name="actionSaveScreenshot"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionE_xit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSpreadsheet"/>
    <addaction name="actionMeasurement_Grid"/>
    <addaction name="actionShowRPM"/>
    <addaction name="actionGrid_Properties"/>
    <addaction name="separator"/>
    <addaction name="actionChoose_Calibration_File"/>
    <addaction name="actionCropReturns"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPython_Console"/>
    <addaction name="actionShowErrorDialog"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <action name="actionResetDefaultSettings">
     <property name="text">
      <string>Reset LidarView settings</string>
     </property>
     <property name="iconText">
      <string>This will reset all LidarView settings by default</string>
     </property>
    </action>
    <addaction name="actionAbout_LidarView"/>
    <addaction name="actionHelpSupport"/>
    <addaction name="actionHelpSlam"/>
    <addaction name="separator"/>
    <addaction name="actionResetDefaultSettings"/>
    <addaction name="actionAdvanceFeature"/>
   </widget>
   <widget class="QMenu" name="menuViews">
    <property name="title">
     <string>Views</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuFilters">
    <property name="title">
     <string>Filters</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSources">
    <property name="title">
     <string>Sources</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdvance">
    <property name="title">
     <string>Advance</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuTools"/>
   <addaction name="menuSources"/>
   <addaction name="menuFilters"/>
   <addaction name="menuViews"/>
   <addaction name="menuHelp"/>
   <addaction name="menuAdvance"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Basic Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpenPcap"/>
   <addaction name="actionOpen_Sensor_Stream"/>
   <addaction name="actionSaveLidarState"/>
   <addaction name="actionLoadLidarState"/>
   <addaction name="actionChoose_Calibration_File"/>
   <addaction name="actionCropReturns"/>
   <addaction name="separator"/>
   <addaction name="actionSelect_Points"/>
   <addaction name="actionSpreadsheet"/>
   <addaction name="actionPython_Console"/>
   <addaction name="actionShowErrorDialog"/>
   <addaction name="actionEnableAdvancedArrays"/>
  </widget>
  <widget class="lqColorToolbar" name="colorToolBar">
   <property name="windowTitle">
    <string>Color Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="pqCustomViewpointsToolbar" name="Custom Viewpoints Toolbar">
   <property name="windowTitle">
    <string>Custom Viewpoints</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QToolBar" name="viewSettings">
   <property name="windowTitle">
    <string>View Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionToggleProjection"/>
   <addaction name="actionMeasure"/>
   <addaction name="actionPlaneFit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QToolBar" name="geolocationToolbar">
   <property name="windowTitle">
    <string>Geolocation Controls</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShowPosition"/>
  </widget>
  <widget class="QDockWidget" name="colorMapEditorDock">
   <property name="windowTitle">
    <string>Color Map Editor</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqColorMapEditor" name="colorMapEditorPanel"/>
  </widget>
  <widget class="QDockWidget" name="pipelineBrowserDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Pipeline Browser</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPipelineBrowserWidget" name="pipelineBrowser"/>
  </widget>
  <widget class="QDockWidget" name="propertiesDock">
   <property name="features">
    <set>QDockWidget::AllDockWidgetFeatures</set>
   </property>
   <property name="windowTitle">
    <string>Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="propertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="memoryInspectorDock">
   <property name="windowTitle">
    <string>Memory Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqMemoryInspectorPanel" name="memoryInspectorPanel"/>
  </widget>
  <widget class="QDockWidget" name="informationDock">
   <property name="windowTitle">
    <string>Information</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="pqProxyInformationWidget" name="proxyInformationWidget"/>
  </widget>
  <widget class="QDockWidget" name="displayPropertiesDock">
   <property name="windowTitle">
    <string>Display</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="displayPropertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="viewPropertiesDock">
   <property name="windowTitle">
    <string>View</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPropertiesPanel" name="viewPropertiesPanel"/>
  </widget>
  <widget class="QDockWidget" name="viewAnimationDock">
   <property name="windowTitle">
    <string>Animation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="pqPVAnimationWidget" name="animationPanel"/>
  </widget>
  <widget class="QDockWidget" name="outputWidgetDock">
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Output Messages</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqOutputWidget" name="outputWidget">
    <property name="settingsKey" stdset="0">
     <string>OutputMessages</string>
    </property>
   </widget>
  </widget>
  <widget class="QDockWidget" name="pythonShellDock">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Python Shell</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="pythonShellDummy">
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Python support not available!</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionE_xit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="statusTip">
    <string>Exit Application</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionOpenPcap">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/WiresharkDoc-128.png</normaloff>:/LidarViewPlugin/WiresharkDoc-128.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Capture File...</string>
   </property>
   <property name="toolTip">
    <string>Open a Wireshark-format capture file with data collected from a lidar sensor</string>
   </property>
   <property name="statusTip">
    <string>Open Data File</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSelect_Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqFrustumSelectionPoint24.png</normaloff>:/vvResources/Icons/pqFrustumSelectionPoint24.png</iconset>
   </property>
   <property name="text">
    <string>Select Points</string>
   </property>
   <property name="toolTip">
    <string>Select Points</string>
   </property>
  </action>
  <action name="actionSpreadsheet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqSpreadsheet16.png</normaloff>:/vvResources/Icons/pqSpreadsheet16.png</iconset>
   </property>
   <property name="text">
    <string>Spreadsheet</string>
   </property>
   <property name="toolTip">
    <string>Spreadsheet</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="actionOpen_Sensor_Stream">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/lidar.png</normaloff>:/LidarViewPlugin/lidar.png</iconset>
   </property>
   <property name="text">
    <string>Sensor &amp;Stream...</string>
   </property>
  </action>
  <action name="actionSaveLidarState">
   <property name="icon">
    <iconset resource="../LVCore/ApplicationComponents/lqResources.qrc">
     <normaloff>:/lqResources/Icons/pqSaveState.png</normaloff>:/lqResources/Icons/pqSaveState.png</iconset>
   </property>
   <property name="text">
    <string>Save Lidar state</string>
   </property>
  </action>
  <action name="actionLoadLidarState">
   <property name="icon">
    <iconset resource="../LVCore/ApplicationComponents/lqResources.qrc">
     <normaloff>:/lqResources/Icons/pqLoadState.png</normaloff>:/lqResources/Icons/pqLoadState.png</iconset>
   </property>
   <property name="text">
    <string>Load Lidar state</string>
   </property>
  </action>
  <action name="actionChoose_Calibration_File">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/calibrate.png</normaloff>:/LidarViewPlugin/calibrate.png</iconset>
   </property>
   <property name="text">
    <string>Choose &amp;Calibration File...</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/media-playback-pause.png</normaloff>:/LidarViewPlugin/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionPython_Console">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/python_logo_124.png</normaloff>:/vvResources/Icons/python_logo_124.png</iconset>
   </property>
   <property name="text">
    <string>Python Console</string>
   </property>
   <property name="shortcut">
    <string>F8</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>&amp;Close Data</string>
   </property>
   <property name="toolTip">
    <string>Close Data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionMeasurement_Grid">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Measurement Grid</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+G</string>
   </property>
  </action>
  <action name="actionSaveScreenshot">
   <property name="text">
    <string>Save &amp;Screenshot...</string>
   </property>
  </action>
  <action name="actionSaveCSV">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/csv.png</normaloff>:/LidarViewPlugin/csv.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;CSV...</string>
   </property>
  </action>
  <action name="actionGrid_Properties">
   <property name="text">
    <string>Grid Properties</string>
   </property>
  </action>
  <action name="actionSavePCAP">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/WiresharkDoc-128.png</normaloff>:/LidarViewPlugin/WiresharkDoc-128.png</iconset>
   </property>
   <property name="text">
    <string>Save PCAP...</string>
   </property>
   <property name="iconText">
    <string>Save PCAP</string>
   </property>
   <property name="toolTip">
    <string>Save PCAP</string>
   </property>
  </action>
  <action name="actionSavePCD">
   <property name="text">
    <string>Save PCD...</string>
   </property>
   <property name="iconText">
    <string>Save PCD</string>
   </property>
   <property name="toolTip">
    <string>Save PCD</string>
   </property>
  </action>
  <action name="actionSavePLY">
   <property name="text">
    <string>Save PLY...</string>
   </property>
   <property name="iconText">
    <string>Save PLY</string>
   </property>
   <property name="toolTip">
    <string>Save PLY</string>
   </property>
  </action>
  <action name="actionSaveLAS">
   <property name="text">
    <string>Save LAS...</string>
   </property>
   <property name="iconText">
    <string>Save LAS</string>
   </property>
   <property name="toolTip">
    <string>Save LAS</string>
   </property>
  </action>
  <action name="actionAbout_LidarView">
   <property name="text">
    <string>About ...</string>
   </property>
  </action>
  <action name="actionHelpSupport">
   <property name="text">
    <string>Professional Support</string>
   </property>
  </action>
  <action name="actionHelpSlam">
   <property name="text">
    <string>How to SLAM</string>
   </property>
  </action>
  <action name="actionRecent_file1">
   <property name="text">
    <string>recent_file1</string>
   </property>
  </action>
  <action name="actionRecent_file2">
   <property name="text">
    <string>recent_file2</string>
   </property>
  </action>
  <action name="actionRecent_file3">
   <property name="text">
    <string>recent_file3</string>
   </property>
  </action>
  <action name="actionRecent_file4">
   <property name="text">
    <string>recent_file4</string>
   </property>
  </action>
  <action name="actionRecent_file5">
   <property name="text">
    <string>recent_file5</string>
   </property>
  </action>
  <action name="actionClear_Menu">
   <property name="text">
    <string>Clear Menu</string>
   </property>
  </action>
  <action name="actionToggleProjection">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>:/vvResources/Icons/ViewOrtho.png</normaloff>
     <normalon>:/vvResources/Icons/ViewPerspective.png</normalon>:/vvResources/Icons/ViewOrtho.png</iconset>
   </property>
   <property name="text">
    <string>toggleProjection</string>
   </property>
   <property name="toolTip">
    <string>Toggle between projective and orthogonal view</string>
   </property>
  </action>
  <action name="actionMeasure">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/Measurements.png</normaloff>:/vvResources/Icons/Measurements.png</iconset>
   </property>
   <property name="text">
    <string>Advanced Selection and Measurement options</string>
   </property>
   <property name="toolTip">
    <string>Advanced Selection and Measurement options</string>
   </property>
  </action>
  <action name="actionPlaneFit">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/planefit.png</normaloff>:/vvResources/Icons/planefit.png</iconset>
   </property>
   <property name="text">
    <string>Plane Fit</string>
   </property>
   <property name="toolTip">
    <string>Plane Fit to selection</string>
   </property>
  </action>
  <action name="actionSavePositionCSV">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/pqSpreadsheet16.png</normaloff>:/vvResources/Icons/pqSpreadsheet16.png</iconset>
   </property>
   <property name="text">
    <string>Save Position CSV...</string>
   </property>
  </action>
  <action name="actionGPSApply">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/location.png</normaloff>:/vvResources/Icons/location.png</iconset>
   </property>
   <property name="text">
    <string>GPSApply</string>
   </property>
   <property name="toolTip">
    <string>Apply geolocation</string>
   </property>
  </action>
  <action name="actionAbsolute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/absolute.png</normaloff>:/vvResources/Icons/absolute.png</iconset>
   </property>
   <property name="text">
    <string>absolute</string>
   </property>
   <property name="toolTip">
    <string>Geolocation in absolute coordinates</string>
   </property>
  </action>
  <action name="actionSaveLAS">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/las.png</normaloff>:/LidarViewPlugin/las.png</iconset>
   </property>
   <property name="text">
    <string>Save &amp;LAS</string>
   </property>
   <property name="toolTip">
    <string>Save to ASPRS LAS format</string>
   </property>
  </action>
  <action name="actionCropReturns">
   <property name="icon">
    <iconset>
     <normaloff>:/LidarViewPlugin/crop.png</normaloff>:/LidarViewPlugin/crop.png</iconset>
   </property>
   <property name="text">
    <string>Crop Returns...</string>
   </property>
  </action>
  <action name="actionShowRPM">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show FrameRate</string>
   </property>
   <property name="toolTip">
    <string>Display the rotations per minute</string>
   </property>
  </action>
  <action name="actionPersisting_Grid_Properties">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Persisting Grid Properties</string>
   </property>
  </action>
  <action name="actionShowErrorDialog">
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/errorConsole.png</normaloff>:/vvResources/Icons/errorConsole.png</iconset>
   </property>
   <property name="text">
    <string>Error Console</string>
   </property>
   <property name="toolTip">
    <string>Show the error console.</string>
   </property>
  </action>
  <action name="actionEnableAdvancedArrays">
   <property name="icon">
    <iconset resource="../LVCore/ApplicationComponents/lqResources.qrc">
     <normaloff>:/lqResources/Icons/EnableAdvancedArrays.png</normaloff>:/lqResources/Icons/EnableAdvancedArrays.png</iconset>
   </property>
   <property name="text">
    <string>Enable Advanced Arrays</string>
   </property>
   <property name="toolTip">
    <string>Enable the interpreter's advanced arrays.</string>
   </property>
  </action>
  <action name="actionAdvanceFeature">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Advance Feature</string>
   </property>
  </action>
  <action name="actionShowPosition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="vvResources.qrc">
     <normaloff>:/vvResources/Icons/map.png</normaloff>:/vvResources/Icons/map.png</iconset>
   </property>
   <property name="text">
    <string>actionShowPosition</string>
   </property>
   <property name="toolTip">
    <string>Show the GPS Track</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>pqCustomViewpointsToolbar</class>
   <extends>QToolBar</extends>
   <header>pqCustomViewpointsToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>lqColorToolbar</class>
   <extends>QToolBar</extends>
   <header>lqColorToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>pqColorMapEditor</class>
   <extends>QWidget</extends>
   <header>pqColorMapEditor.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPipelineBrowserWidget</class>
   <extends>QWidget</extends>
   <header>pqPipelineBrowserWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>pqPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqProxyInformationWidget</class>
   <extends>QWidget</extends>
   <header>pqProxyInformationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqMemoryInspectorPanel</class>
   <extends>QWidget</extends>
   <header>pqMemoryInspectorPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqPVAnimationWidget</class>
   <extends>QWidget</extends>
   <header>pqPVAnimationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqOutputWidget</class>
   <extends>QWidget</extends>
   <header>pqOutputWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="vvResources.qrc"/>
  <include location="../LVCore/ApplicationComponents/lqResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionE_xit</sender>
   <signal>triggered()</signal>
   <receiver>vvMainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
